generators/firechip/src/main/scala/Generator.scala

```diff
@@ -0,0 +1,133 @@
+//See LICENSE for license details.
```

Comment:
This file seems to be executable. `-rwxrwxr-x`. Can you confirm if this is the case on your end. And if so, can we switch it to something like `-rw-rw-r--`

Reply:
As below.